Content.Client/PipePainter/PipePainterBoundUserInterface.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Content.Client.PipePainter.UI;
+using Content.Shared.PipePainter;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.PipePainter;
+
+public sealed class PipePainterBoundUserInterface : BoundUserInterface
+{
+    private PipePainterWindow? _window;
+
+    public PipePainterBoundUserInterface(ClientUserInterfaceComponent owner, Enum uiKey) : base(owner, uiKey)
```

Comment:
```suggestion
    public PipePainterBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner, uiKey)
```

Now EntityUid is required as the owner.